sys/net/gnrc/netif/ieee802154/gnrc_netif_ieee802154.c

```diff
@@ -229,6 +246,11 @@ static int _send(gnrc_netif_t *netif, gnrc_pktsnip_t *pkt)
     uint8_t flags = (uint8_t)(state->flags & NETDEV_IEEE802154_SEND_MASK);
     le_uint16_t dev_pan = byteorder_btols(byteorder_htons(state->pan));
 
+    /* send raw 802.15.4 packets out without adding a header */
+    if (netif->flags & GNRC_NETIF_FLAGS_RAWMODE) {
+        return _send_raw(netif, (iolist_t *)pkt);
+    }
```

Comment:
Does it make sense to use `gnrc_netif` like this? (@miri64)
It seems like you don't want to use GNRC at all, but ZBOSS instead.

Will the new radio HAL help here @jia200x?

Reply:
Probably. Does this ZBOSS stack require direct access to the radio? If yes, that should be the way to go.